README.md

```diff
@@ -15,14 +15,15 @@ The state-aware workflow will look for the `manifest.json` file in a branch call
 ## How To Set Up Your dbt Project
 
  1. Fork this repo and copy your whole dbt project into the `project_goes_here` folder.
- 2. Update your repository settings to allow GitHub Actions to create PRs. This setting can be found in a repository's settings under Actions > General > Workflow permissions. [This is what it should look like.](https://user-images.githubusercontent.com/21294829/263915123-512bf335-6796-4ae3-a7dc-ad1cf6c4035f.png)
- 3. Go to the Actions tab and run the `Project Setup` workflow, making sure to select the type of database you want to set up
+ 2. Create a [Personal Access Token](https://github.com/settings/tokens?type=beta) with Workflows (Read/Write) permission and add it to the repository action secrets with key WORKFLOW_TOKEN
```

Comment:
@jayashe this could be specific to your org setup, maybe you can reword this to specify this is an optional step depending on how ownership/admin of the target repo is handled